### PR TITLE
Implement adapters for inspections

### PR DESCRIPTION
**Pull request by fridex**
# This introduces a breaking change

- [x] Yes
- [ ] No

This will break things on metrics-exporter side.

### CODE REVIEW

**Comment by fridex** [comment]
recheck

File: thoth/storages/inspections.py [hunk 1]
```diff
@@ -17,37 +17,144 @@
...
+    def connect(self) -> None:
+        """Connect this adapter to Ceph."""
+        self.ceph.connect()
+
+    def is_connected(self) -> bool:
+        """Check if this adapter is connected."""
+        return self.ceph.is_connected()
+
+    def check_connection(self) -> None:
+        """Check connection of this adapter.."""
```
Comment (pacospace) [inline thoth/storages/inspections.py:55]: One .

Comment (fridex) [reply thoth/storages/inspections.py]: Fixed, PTAL.

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
regate

**Pull request merged**